src/protocol/resp/src/request/badd.rs

```diff
@@ -0,0 +1,144 @@
+// Copyright 2022 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+use super::*;
+use std::io::{Error, ErrorKind};
+use std::sync::Arc;
+
+type ArcByteSlice = Arc<Box<[u8]>>;
+type ArcKeyValuePair = (ArcByteSlice, ArcByteSlice);
+
+/// Represents the btree add command which was added to Twitter's internal version of redis32.
```

Comment:
Keep doc lines to 80 characters width maximum to match the rest of the codebase. Split this across multiple-lines and it'll be fine.